app/secrets.py

```diff
@@ -15,8 +13,44 @@
     "EQ_SERVER_SIDE_STORAGE_DATABASE_PASSWORD",
 ]
 
+PURPOSE_AUTHENTICATION = 'authentication'
+PURPOSE_SUBMISSION = 'submission'
```

Comment:
I think it would be clearer in the places these are used if they were `KEY_PURPOSE_AUTHENTICATION` and `KEY_ PURPOSE_SUBMISSION`